hw/ip/hmac/dv/hmac_sim_cfg.hjson

```diff
@@ -110,6 +110,7 @@
     {
       name: smoke
       tests: ["hmac_smoke", "hmac_test_sha_vectors"]
+      run_opts: ["+smoke_test=1"]
```

Comment:
Awesome job adding this here! This means those test specifications need not be duplicated. Could you replicate this for `rv_timer` as well?

+ @udinator for reference - you could make similar change in your PR.

Reply:
Sure, I can do that for rv_timer!